frontend/projects/upgrade/src/app/features/dashboard/home/components/metrics/metrics.component.ts

```diff
@@ -492,7 +497,6 @@ export class MonitoredMetricsComponent implements OnInit, OnChanges, OnDestroy {
                     : ({...this.removeMetricName(queryObj)})
                   );
               } else {
```

Comment:
I don't think these `else` blocks are doing anything, can they both be removed?

Reply:
correct, I will update

Reply:
correct